xla/service/cpu/onednn_contraction_rewriter.cc

```diff
@@ -400,8 +400,11 @@ absl::StatusOr<Shape> AdjustAddendShape(const HloInstruction* contraction,
   // If rank(new_shape) > rank(instr), extra dimensions with value = 1 can be
   // deleted from the new_shape.
   auto instr_shape = contraction->shape();
-  int64_t rank_difference =
-      new_shape.dimensions().size() - instr_shape.dimensions().size();
+  int64_t rank_difference = 0;
+  if (new_shape.dimensions().size() > instr_shape.dimensions().size()) {
+    rank_difference =
+        new_shape.dimensions().size() - instr_shape.dimensions().size();
+  }
```

Comment:
Nit: This can be condensed to 
```suggestion
  int64_t rank_difference = max(0, new_shape.dimensions().size() - instr_shape.dimensions().size());
```

(Need reformatting.)

Reply:
Done